src/averageAges.js

```diff
@@ -37,7 +47,14 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const arrayOfWomen = withChildren
+    ? people.filter(person => person.sex === 'f'
```

Comment:
Consider using more descriptive variable names, like `womenArray` instead of `arrayOfWomen`. 🤔